micah-akpan/questioner-react
# #164695740 Complete set-up of project for react development

#### What does this PR do?
It set-ups the React environment to aid the development of the project's frontend
#### Description of Task to be completed?
Set up React project, using tools like webpack for module bundling
#### How should this be manually tested?
Clone repo, then `cd` into it then:
- Run `npm install`
- Run `npm run dev` or `yarn dev`
- Your default browser will open a new tab and you should see a warm greeting to the app
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[164695740](https://www.pivotaltracker.com/story/show/164695740)
#### Screenshots (if appropriate)
#### Questions:
